02-intermediate-ruby/api-testing-vcr.md

```diff
@@ -20,6 +20,23 @@ Each interaction is recorded as a **cassette**. We can load cassettes in our tes
 
 ![VCR](images/vcr.jpg)
 
+### What is VCR doing?
+VCR is essentially a recording device that interrupts the normal way they we send messages out to the internet. The first time a set of tests that use VCR run, the cassette takes note of the new URL:
+
+![A VHS cassette watches a message leave HTTParty, taking note](images/apis/VCR_step_1.png "A VHS cassette watches a message leave HTTParty, taking note")
+
+When the API responds, the cassette takes note of the contents of the response as well:
+
+![A VHS cassette sees the response from the API before it gets back to HTTParty.](images/apis/VCR_step_2.png "A VHS cassette sees the response from the API before it gets back to HTTParty.")
+
+Whenever that URL is sent out in the future, VCR gobbles up the message before it goes to the API, taking the API out of the equation:
```

Comment:
Suggestion: Whenever the same request (matching on request method, URI, and body) is sent out in the future, ...